javascripts/govuk/selection-buttons.js

```diff
@@ -83,7 +83,9 @@
   }
   SelectionButtons.prototype.getClickHandler = function () {
     return function (e) {
-      this.markSelected($(e.target))
+      var target = $(e.target)
+      this.markSelected(target)
+      target.trigger('focus')
```

Comment:
Well, the other thing is that eventually I’d like to get rid of this JS entirely with a CSS solution, which would mean that the behaviour will revert to track whatever the native versions do.